src/components/Photo/DocumentAutoCapture.js

```diff
@@ -39,9 +31,10 @@ export default class DocumentAutoCapture extends Component<Props, State> {
 
   interval: ?Visibility
 
+  captureIds: string[] = []
```

Comment:
We were also storing `captures` in state, which cause re-renders on each change
This was unnecessary, and actually has a memory overhead too (since React needs to compare a load of VDOM stuff), so changing this to be just a regular ol' instance variable saves a fair bit of memory